src/compas_rhino/scene/meshobject.py

```diff
@@ -264,7 +200,9 @@ def draw_vertices(self):
             elif self.group:
                 self.add_to_group(self.group, guids)
 
-        self._guids_vertices = guids
+        self._guid_vertex = dict(zip(guids, vertices))
+
+        self._guids += guids
```

Comment:
This causes an error if the scene has not been drawn at least once because `self._guids` is `None`. Maybe should be initialized on `__init__`

Reply:
ha weird maybe i forgot to push this

Reply:
shoud be fixed now